diff -up bino-1.2.1/src/Makefile.in.orig bino-1.2.1/src/Makefile.in
--- bino-1.2.1/src/Makefile.in.orig	2011-10-10 21:18:59.000000000 +0200
+++ bino-1.2.1/src/Makefile.in	2011-11-09 19:54:08.419550089 +0100
@@ -445,7 +445,7 @@ AM_CPPFLAGS = -I$(top_srcdir)/src/base $
 bino_LDADD = $(top_builddir)/src/base/libbase.a $(LIBINTL) $(LIBICONV) \
 	$(libavformat_LIBS) $(libavdevice_LIBS) $(libswscale_LIBS) \
 	$(libass_LIBS) $(libopenal_LIBS) $(libgl_LIBS) $(libglew_LIBS) \
-	$(libqtopengl_LIBS) $(am__append_3) $(am__append_6)
+	$(libqtopengl_LIBS) $(am__append_3) $(am__append_6) -lGLEW -lGLEWmx
 iconsdatadir = $(datadir)/icons/hicolor
 icons16dir = $(iconsdatadir)/16x16/apps
 icons32dir = $(iconsdatadir)/32x32/apps
