diff -up bino-1.2.1/src/Makefile.am.orig bino-1.2.1/src/Makefile.am
--- bino-1.2.1/src/Makefile.am.orig	2011-10-01 09:00:55.000000000 +0200
+++ bino-1.2.1/src/Makefile.am	2012-03-01 14:36:22.119185909 +0100
@@ -56,21 +56,21 @@ ICONS_LOCAL_IPE = \
 	icons-local/output-type-stereo.ipe
 
 ICONS = \
-	icons/application-exit.png \
-	icons/applications-internet.png \
-        icons/camera-web.png \
-	icons/document-open.png \
-	icons/help-about.png \
-	icons/help-contents.png \
-	icons/media-playback-pause.png \
-	icons/media-playback-start.png \
-	icons/media-playback-stop.png \
-	icons/media-playlist-repeat.png \
-	icons/media-seek-backward.png \
-	icons/media-seek-forward.png \
-	icons/preferences-desktop-keyboard.png \
-	icons/view-fullscreen.png \
-	icons/view-restore.png \
+	/usr/share/icons/oxygen/48x48/actions/application-exit.png \
+	/usr/share/icons/oxygen/48x48/categories/applications-internet.png \
+	/usr/share/icons/oxygen/48x48/devices/camera-web.png \
+	/usr/share/icons/oxygen/48x48/actions/document-open.png \
+	/usr/share/icons/oxygen/48x48/actions/help-about.png \
+	/usr/share/icons/oxygen/48x48/actions/help-contents.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-pause.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-start.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-stop.png \
+	/usr/share/icons/oxygen/48x48/status/media-playlist-repeat.png \
+	/usr/share/icons/oxygen/48x48/actions/media-seek-backward.png \
+	/usr/share/icons/oxygen/48x48/actions/media-seek-forward.png \
+	/usr/share/icons/oxygen/48x48/apps/preferences-desktop-keyboard.png \
+	/usr/share/icons/oxygen/48x48/actions/view-fullscreen.png \
+	/usr/share/icons/oxygen/48x48/actions/view-restore.png \
 	$(patsubst %.ipe,%.png,$(ICONS_LOCAL_IPE))
 
 EXTRA_DIST = \
@@ -92,7 +92,7 @@ qt_resources.qrc:
 	echo '<file alias="LICENSE">$(top_srcdir)/COPYING</file>'; \
 	echo '<file alias="logo/bino_logo.png">$(top_srcdir)/src/logo/bino_logo.png</file>'; \
 	echo '<file alias="logo/64x64/bino.png">$(top_srcdir)/src/logo/64x64/bino.png</file>'; \
-	for icon in $(ICONS); do echo '<file alias="'"$${icon}"'">$(top_srcdir)/src/'"$${icon}"'</file>'; done; \
+	for icon in $(ICONS); do echo '<file alias="'"$${icon}"'">'"$${icon}"'</file>'; done; \
 	echo '</qresource>'; \
 	echo '</RCC>') > $@
 
diff -up bino-1.2.1/src/Makefile.in.orig bino-1.2.1/src/Makefile.in
--- bino-1.2.1/src/Makefile.in.orig	2011-10-10 21:18:59.000000000 +0200
+++ bino-1.2.1/src/Makefile.in	2012-03-01 14:36:35.726961839 +0100
@@ -399,21 +399,21 @@ ICONS_LOCAL_IPE = \
 	icons-local/output-type-stereo.ipe
 
 ICONS = \
-	icons/application-exit.png \
-	icons/applications-internet.png \
-        icons/camera-web.png \
-	icons/document-open.png \
-	icons/help-about.png \
-	icons/help-contents.png \
-	icons/media-playback-pause.png \
-	icons/media-playback-start.png \
-	icons/media-playback-stop.png \
-	icons/media-playlist-repeat.png \
-	icons/media-seek-backward.png \
-	icons/media-seek-forward.png \
-	icons/preferences-desktop-keyboard.png \
-	icons/view-fullscreen.png \
-	icons/view-restore.png \
+	/usr/share/icons/oxygen/48x48/actions/application-exit.png \
+	/usr/share/icons/oxygen/48x48/categories/applications-internet.png \
+	/usr/share/icons/oxygen/48x48/devices/camera-web.png \
+	/usr/share/icons/oxygen/48x48/actions/document-open.png \
+	/usr/share/icons/oxygen/48x48/actions/help-about.png \
+	/usr/share/icons/oxygen/48x48/actions/help-contents.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-pause.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-start.png \
+	/usr/share/icons/oxygen/48x48/actions/media-playback-stop.png \
+	/usr/share/icons/oxygen/48x48/status/media-playlist-repeat.png \
+	/usr/share/icons/oxygen/48x48/actions/media-seek-backward.png \
+	/usr/share/icons/oxygen/48x48/actions/media-seek-forward.png \
+	/usr/share/icons/oxygen/48x48/apps/preferences-desktop-keyboard.png \
+	/usr/share/icons/oxygen/48x48/actions/view-fullscreen.png \
+	/usr/share/icons/oxygen/48x48/actions/view-restore.png \
 	$(patsubst %.ipe,%.png,$(ICONS_LOCAL_IPE))
 
 EXTRA_DIST = \
@@ -1053,7 +1053,7 @@ qt_resources.qrc:
 	echo '<file alias="LICENSE">$(top_srcdir)/COPYING</file>'; \
 	echo '<file alias="logo/bino_logo.png">$(top_srcdir)/src/logo/bino_logo.png</file>'; \
 	echo '<file alias="logo/64x64/bino.png">$(top_srcdir)/src/logo/64x64/bino.png</file>'; \
-	for icon in $(ICONS); do echo '<file alias="'"$${icon}"'">$(top_srcdir)/src/'"$${icon}"'</file>'; done; \
+	for icon in $(ICONS); do echo '<file alias="'"$${icon}"'">'"$${icon}"'</file>'; done; \
 	echo '</qresource>'; \
 	echo '</RCC>') > $@
 
